python/lsst/ctrl/bps/htcondor/htcondor_service.py

```diff
@@ -1207,20 +1223,20 @@ def _create_detailed_report_from_jobs(wms_workflow_id, jobs):
         jobs=[],
         total_number_jobs=dag_job["total_jobs"],
         job_state_counts=dag_job["state_counts"],
+        exit_code_summary=_get_exit_code_summary(task_jobs),
     )
 
-    for job_id, job_info in jobs.items():
+    for job_id, job_info in task_jobs.items():
         try:
-            if job_info["ClusterId"] != int(float(wms_workflow_id)):
-                job_report = WmsJobReport(
-                    wms_id=job_id,
-                    name=job_info.get("DAGNodeName", job_id),
-                    label=job_info.get("bps_job_label", pegasus_name_to_label(job_info["DAGNodeName"])),
-                    state=_htc_status_to_wms_state(job_info),
-                )
-                if job_report.label == "init":
-                    job_report.label = "pipetaskInit"
-                report.jobs.append(job_report)
+            job_report = WmsJobReport(
+                wms_id=job_id,
+                name=job_info.get("DAGNodeName", job_id),
+                label=job_info.get("bps_job_label", pegasus_name_to_label(job_info["DAGNodeName"])),
```

Comment:
Some day we should check if there's a better name for "pegasus_name_to_label".  I had to go check whether this was dependent on the no-longer-maintained pegasus plugin package (it isn't at least at the import level).

Reply:
Just to be clear, this snippet showed up in this pull request only because I needed to adjust its indentation due to the changes I made near it. I didn't modify this fragment in any manner. So while I agree with the sentiment I'd prefer to address this issue in a separate ticket.